src/System.Data.Common/tests/System/Data/DataTableTest.cs

```diff
@@ -3776,12 +3776,12 @@ public void WriteXmlSchema_ForignKeyConstraint()
             StringWriter sw1 = new StringWriter();
             ds1.Tables[0].WriteXmlSchema(sw1);
             string xml1 = sw1.ToString();
-            Assert.True(xml1.IndexOf(@"<xs:unique name=""Constraint1"">") != -1);
+            Assert.True(xml1.Contains(@"<xs:unique name=""Constraint1"">"));
 
             StringWriter sw2 = new StringWriter();
             ds1.Tables[1].WriteXmlSchema(sw2);
             string xml2 = sw2.ToString();
-            Assert.True(xml2.IndexOf(@"<xs:unique name=""Constraint1"">") == -1);
+            Assert.False(xml2.Contains(@"<xs:unique name=""Constraint1"">"));
```

Comment:
There is no `AssertExtensions.DoesNotContain` but you can use `Assert.DoesNotContain(@"<xs:unique name=""Constraint1"">", xml2,  StringComparison.Ordinal)`

What you have is fine, too

Reply:
Or feel free to add `AssertExtensions.DoesNotContain`  in AssertExtensions.cs if you like.

Reply:
Updated accordingly to your first suggestion.